contrib/vagrant/start.sh

```diff
@@ -284,7 +284,7 @@ if [ -z "${RELOAD}" ]; then
 fi
 
 if [ -n "${VM_PROXY}" ]; then
-    export WGET="HTTPS_PROXY=${VM_PROXY} wget"
+    export WGET="https_proxy=${VM_PROXY} wget"
```

Comment:
I'm surprised that this change was necessary, but cross-referencing with the `wget` manpages this does seem to be correct. I don't know why the original author of this code didn't notice this. :+1: